website/docs/r/generic_secret.html.md

```diff
@@ -58,6 +58,11 @@ The following arguments are supported:
   authentication is not able to read the data. Setting this to `true` will
   break drift detection. Defaults to false.
 
+* `delete_all_versions` - (Optional) true/false.  Only applicable for kv-v2 stores.
```

Comment:
Here `true` is lowercase, but `disable_read` has it capitalized, mind updating the `disable_read` to match?

Reply:
Fixed